bento/src/main/java/com/yelp/android/bento/componentcontrollers/RecyclerViewComponentController.java

```diff
@@ -334,6 +416,20 @@ public void onItemPickedUp(ComponentViewHolder viewHolder) {
         }
     }
 
+    private void addComponentInternal(int index, @NonNull final Component component) {
+        if (index == -1) {
```

Comment:
I think we should compare with INVALID_COMPONENT_INDEX instead of the literal here